Fix documentation build errors with Perl 5.18 pod2man

This fixes errors building man pages with newer versions of pod2man
included with Perl 5.18.

Most was fixed in 1.0.1h. Only 1 left :)

Upstream-Status: Submitted
Signed-off-by: Jonathan Liu

Index: openssl-1.0.1f/doc/ssl/SSL_shutdown.pod
===================================================================
--- openssl-1.0.1f.orig/doc/ssl/SSL_shutdown.pod	2014-06-05 12:41:31.000000000 +0300
+++ openssl-1.0.1f/doc/ssl/SSL_shutdown.pod	    2014-06-09 12:11:08.896622663 +0300
@@ -104,7 +104,7 @@
 The shutdown was successfully completed. The "close notify" alert was sent
 and the peer's "close notify" alert was received.
 
-=item -1
+=item Z<>-1
 
 The shutdown was not successful because a fatal error occurred either
 at the protocol level or a connection failure occurred. It can also occur if
